modules/fb_instant_articles_display/fb_instant_articles_display.module

```diff
@@ -6,6 +6,49 @@
  */
 
 /**
+* Implements hook_init().
+ *
+ * @todo Update to use a custom DrupalFBInstantArticlesLogger once
+ *   @link https://github.com/facebook/facebook-instant-articles-sdk-php/issues/14 this SDK issue @endlink is in resolved.
+*/
+function fb_instant_articles_display_init() {
```

Comment:
Should this logger be configured on hook_init or only whenever the Transformer is being used?